styles.css

```diff
@@ -115,14 +115,20 @@ h1 {
 .right-side {
   background-color: #d0d2d3;
   width: 50%;
+  border-left: 2px solid #BFBFBF;
+  display: flex;
+  justify-content: flex-start;
+  padding: .75% 0;
+  overflow-x: scroll;
 }
 
 .game-box {
   background-color: #FFFFFF;
   width: 89%;
   padding: 2%;
-  border: 1px solid #BFBFBF;
```

Comment:
Changed border color for left side game-boxes to lighter to match comp used darker color  later on winner-card